src/openpgp.js

```diff
@@ -369,6 +369,7 @@ export async function decrypt({ message, decryptionKeys, passwords, sessionKeys,
         result.data,
         stream.fromAsync(async () => {
           await util.anyPromise(result.signatures.map(sig => sig.verified));
+          if (format === 'binary') return new Uint8Array();
```

Comment:
I'd have this return `''` explicitly for string streams as well (and separately, I'd consider having the stream helpers be stricter with type checks and actually throw when e.g. joining `undefined` chunks in a stream string, wdyt...?)

Reply:
I sort of agree but at the same time, I think it would be useful to formally support `undefined` in web-stream-tools, and just make them a no-op (ignore them when writing), that way we don't have to concatenate another value. This was just the quickest fix until that's done.
But yeah, until we have that we can return `''` as well.

Reply:
Fine by me, I mostly care about consistent behavior, esp. thinking of the TS transition 🙂